cypress/integration/wireless/create.spec.js

```diff
@@ -0,0 +1,46 @@
+const apiResponses = require('../../fixtures/apiResponses.json')
+const wirlessFixtures = require('../../fixtures/wireless.json')
+
+describe("create a wireless profile", () => {
+    beforeEach("clear cache and login", () => {
+        cy.setup()
+    })
+
+    it("creates a deafault profile", () => {
+        cy.myIntercept("POST", "wirelessconfigs", {
+            statusCode: apiResponses.wirelessConfigs.create.success.code,
+            body: apiResponses.wirelessConfigs.create.success.response,
+        }).as("post-wireless")
+
+        cy.myIntercept("GET", "wirelessconfigs", {
+            statusCode: apiResponses.wirelessConfigs.getAll.empty.code,
+            body: apiResponses.wirelessConfigs.getAll.empty.response,
+        }).as("get-wireless")
+
+        cy.goToPage("Wireless")
+        cy.wait("@get-wireless")
+
+        //change api response
+        cy.myIntercept("GET", "wirelessconfigs", {
+            statusCode: apiResponses.wirelessConfigs.getAll.success.code,
+            body: apiResponses.wirelessConfigs.getAll.success.response,
+        }).as("get-wireless2")
+
+        cy.get("button").contains("Add New").click()
+        cy.enterWirelessInfo(
+            wirlessFixtures.happyPath.profileName,
+            wirlessFixtures.happyPath.ssid,
+            "P@ssw0rd"
```

Comment:
let's move this to the cypress env like the other passwords

Reply:
Added new variable PSKPASSPHRASE to cypress.json